tests/test_cmd_signature.py

```diff
@@ -89,6 +95,36 @@ def test_sig_merge_1_fromfile_picklist(runtmp):
     assert actual_merge_sig.minhash == test_merge_sig.minhash
 
 
+def test_sig_merge_1_fromfile_picklist_gz(runtmp):
+    # test with --from-file and gzipped picklist
+    c = runtmp
+
+    # merge of 47 & 63 should be union of mins
+    sig47 = utils.get_test_data('47.fa.sig')
+    sig63 = utils.get_test_data('63.fa.sig')
+    sig47and63 = utils.get_test_data('47+63.fa.sig')
+
+    from_file = _write_file(runtmp, 'list.txt', [sig47, sig63])
+    picklist = _write_file(runtmp, 'pl.csv',
```

Comment:
ah, so name doesn't matter for output -- writing `pl.csv` but it's gzipped.

Not an issue, just worth mentioning.

Reply:
right, this is just a test function. It matters if you use the built-in `FileOutputCSV`.